node-tests/strip-lines.js

```diff
@@ -44,4 +44,22 @@ describe('stripNewlines', () => {
 			'{{#foo}}bar{{/foo}}'
 		);
 	});
+
+	it('should not join an angle brackets node', () => {
+		const input =
+`<Foo
+	@bar={{qux}}
+	@fuzz=buzz
+>`;
+		const expected =
+`<Foo
+@bar={{qux}}
+@fuzz=buzz
+>`;
```

Comment:
I would add some more tests like this

```hbs
<Foo 
  @bar={{wow}} 
  {{on 'foo' 
    (qwe)
  }} 
  @foo={{wow}}
>
```

Reply:
Done, I added your test and a middle ground one